### ISSUE TITLE
Distillation code

### ISSUE THREAD

**Issue by sukiAme7** [Westlake-AGI-Lab/Distill-Any-Depth#36]
Great work! I wonder when the distillation code will be open sourced. At the same time, I also want to know that this teacher model can be any depth estimation model such as Depth Pro?

**Comment by shuiyued** [comment]
Of course! You can observe that the distillation loss operates at the depth value level, so we can also distill the model similarly to Depth Pro.

We will release the distillation code once our paper is accepted (っ╥╯﹏╰╥c) ."

**Comment by 20XYSong** [comment]
@shuiyued Hi, will the training code be released? I tried to replicate DAv2-small training with DAD-Large as tea-model, but find DAv2-small outputs magnitude continues to increase as training goes. And finally I can't get a good disparity-quality as official-DAv2-small. Is there any training trick for training this ViT network? Or Huge training data is essential?(Hence I only use part of Places365 and SA1B, totally around 50k images)
